src/geometry.rs

```diff
@@ -71,12 +71,27 @@ use crate::{
     conversion::{CoordSeq, CoordType, ToSFCGALGeom},
     errors::get_last_error,
     utils::{
-        _c_string_with_size, _string, check_computed_value, check_nan_value,
+        _string, c_string_with_size, check_computed_value, check_nan_value,
         check_null_prepared_geom, check_predicate, get_raw_bytes,
     },
     Result, ToSFCGAL,
 };
 
+// For review
+use sfcgal_sys::{
+    sfcgal_geometry_alpha_wrapping_3d, sfcgal_geometry_as_stl, sfcgal_geometry_as_stl_file,
```

Comment:
The CI is running against the latest stable sfcgal release (2.1.0) which doesn't seem to contain the following functions (`sfcgal_sys::sfcgal_geometry_as_stl`, `sfcgal_sys::sfcgal_geometry_as_stl_file`, `sfcgal_sys::sfcgal_geometry_get_geometry_n`, `sfcgal_sys::sfcgal_geometry_get_geometry_n`, `sfcgal_sys::sfcgal_geometry_set_geometry_n`).
I tested locally (and checked in the source distribution of sfcgal 2.1.0) and I can't find them either (although they are in the documentation https://sfcgal.gitlab.io/SFCGAL/API/group__capi/, which is probably based on the current source tree of sfcgal).